content/sensu-go/5.16/api/namespaces.md

```diff
@@ -130,7 +165,8 @@ The `/user-namespaces` API endpoint provides HTTP GET access to the namespaces t
 The following example demonstrates a request to the `/user-namespaces` API endpoint, resulting in a JSON array that contains the namespaces the user has access to.
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/enterprise/user-namespaces -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/enterprise/user-namespaces
+-H "Authorization: Bearer $SENSU_TOKEN"
 [
```

Comment:
We should include either a new line and/or denote that everything after this is the example output and not the command that should be ran.

Reply:
@Blue0ctober I added a blank line -- do you think it's ok like that? I looked at our other GET examples, and they generally include the response this way.